src/configuration.c

```diff
@@ -8,7 +8,9 @@
 
 #include <sys/types.h>
 #include <sys/stat.h>
-#include <sys/sysinfo.h>
+#ifdef __linux__
```

Comment:
I think we can do better here: drop the include altogether.

Looking at a bit of git history it was needed back when this functionality used the Linux-specific `get_nprocs_conf()`, but the current `sysconf(_SC_PROCESSORS_ONLN)` only needs `unistd.h`, and is also available on the BSDs.